src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -599,66 +594,49 @@ public ResponseEntity<ExamExerciseStartPreparationStatus> getExerciseStartStatus
      * @param studentExam the student exam to be prepared
      */
     private void prepareStudentExamForConduction(HttpServletRequest request, User currentUser, StudentExam studentExam) {
-        examService.loadQuizExercisesForStudentExam(studentExam);
 
-        // 2nd: mark the student exam as started
-        studentExam.setStarted(true);
-        if (studentExam.getStartedDate() == null) {
-            studentExam.setStartedDate(ZonedDateTime.now());
+        // In case the studentExam is not yet started, a new participation wit a specific initialization date should be created - isStarted uses Boolean
+        if (studentExam.getExam().isTestExam()) {
+            boolean setupTestExamNeeded = studentExam.isStarted() == null || !studentExam.isStarted();
+            if (setupTestExamNeeded) {
+                // Fix startedDate. As the studentExam.startedDate is used to link the participation.initializationDate, we need to drop the ms
+                // (initializationDate is stored with ms)
+                ZonedDateTime startedDate = ZonedDateTime.now().truncatedTo(ChronoUnit.SECONDS);
+
+                // Set up new participations for the Exercises and set initialisationDate to the startedDate
+                studentExamService.setUpTestExamExerciseParticipationsAndSubmissions(studentExam, startedDate);
+
+                // Mark the student exam as started and save it
+                studentExam.setStarted(true);
+                studentExam.setStartedDate(startedDate);
+                studentExamRepository.save(studentExam);
+            }
+        }
+        else {
+            // Mark the student exam as started and save it
+            studentExam.setStarted(true);
+            if (studentExam.getStartedDate() == null) {
+                studentExam.setStartedDate(ZonedDateTime.now());
```

Comment:
In the if block above the start time is truncated to seconds

Reply:
this was on purpose for test exams